python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -447,22 +447,14 @@ def plot_solution_type(self):
 
         self._add_figure(name="solution_type", figure=figure, title="Solution Type")
 
-    def plot_displacement(self):
+    def plot_displacement(self, source, time, solution_type, displacement_enu_m, std_enu_m):
```

Comment:
Suggest adding a leading _ to make this a protected function since it is no longer something would call directly.